paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -213,10 +213,13 @@ def get_mesos_utilization_error(
         self.log.debug(region_pool_utilization_dict)
         free_pool_resources = region_pool_utilization_dict['free']
         total_pool_resources = region_pool_utilization_dict['total']
-        utilization = 1.0 - min([
-            float(float(pair[0]) / float(pair[1]))
-            for pair in zip(free_pool_resources, total_pool_resources)
-        ])
+        used_percs = []
+        for pair in zip(free_pool_resources, total_pool_resources):
+            try:
+                used_percs.append(float(float(pair[0]) / float(pair[1])))
+            except ZeroDivisionError:  # should occur for 0 gpus, which is valid
```

Comment:
if you can easily check for error condition there's no reason to use exceptions:

    for used, total in zip(free_pool_resources, total_pool_resources):
        if math.isclose(total, 0):
            continue
        used_percs.append(float(used)/float(total))